src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -1379,6 +1379,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Function
 
+        ''' <summary>
+        ''' Gets the underlying <see cref="Conversion"/> information for InConversion of <see cref="IArgument"/> that was created from Visual Basic code.
+        ''' </summary>
+        ''' <param name="argument">The argument to get original info from.</param>
+        ''' <returns>The underlying <see cref="Conversion"/> of the InConversion.</returns>
+        ''' <exception cref="ArgumentException">If the <see cref="IArgument"/> was not created from Visual Basic code.</exception>
+        <Extension>
+        Public Function GetInConversion(argument As IArgument) As Conversion
+            Dim basicArgument = TryCast(argument, BaseVisualBasicArgument)
+            If basicArgument IsNot Nothing Then
+                Return basicArgument.InConversionInternal
+            Else
+                Throw New ArgumentException(String.Format(VBResources.IArgumentIsNotVisualBasicArgument,
```

Comment:
>Throw New ArgumentException(String.Format(VBResources.IArgumentIsNotVisualBasicArgument, [](start = 16, length = 88)

Do we have tests for this code path?

Reply:
Nope, don't have tests for this new extension method. Will add one.

Reply:
Added.

Reply:
> Added

Could you point me to the tests? What file they are in? What are the names?